### DISCUSSION TITLE
Facing issue Failed to create session. An unknown server-side error occurred while processing the command. Original error: Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65 xcodebuild error message on mac mini m2

### DISCUSSION THREAD

**Discussion by rohittripathi262** [appium/.github#2] [Q&A]
Hi Guys,

I have been encountering a session creation problem for the past three days. I have provided all the relevant details below for your reference. I would greatly appreciate it if you could examine the information and offer a solution.

java version       17.0.7
Node version     v20.10.0
NPM version      10.2.5
Appium version  2.2.3
**Appium Inspector** 
App Version 2023.11.1
Electron 13.6.9
Node Js 14.16.0

**Error Screenshot**

![image](https://github.com/appium/.github/assets/146321707/62318936-ef7f-4701-94bb-6d9fb1b7d93d)

**Server Log**

[appium-server-logs.pdf](https://github.com/appium/.github/files/13624069/appium-server-logs.pdf)

**IOS 13 (17.1.2) Screenshot**
Integration App got installed and verified by Device Management 

![image](https://github.com/appium/.github/assets/146321707/fe1de424-998c-447c-a1f1-ffa12d1c4ead)

**What has been tried** 
made a build through this path /Applications/Appium\ Server\ GUI\ 4729.app/Contents/Resources/app/node_modules/appium/node_modules/appium-webdriveragent/WebDriverAgent.xcodeproj

**Comment by rohittripathi262** [comment]
Thanks guys issue got resolved

**Reply by SankarKK** [reply]
Hi Rohit,

I am also facing the same issue now. App is not launching and getting status 500 code error. Could you please help me to resolve this error.

@triager if it possible could you please help me.

**Reply by tatanota** [reply]
xcodebuild failed with code 65.のエラーであればこちらの記事を参照ください
https://qiita.com/BubbleBubble/items/6735647371bf32ddb3b3
XCodeでの設定や、コマンド等の実施でうまくいくかもしれません

「>xcodebuild -project WebDriverAgent.xcodeproj -scheme WebDriverAgentRunner -destination 'id=【UDID】' -allowProvisioningUpdates test」
